src/emulator/controller.ts

```diff
@@ -374,6 +375,52 @@ export async function startAll(options: any, noUi: boolean = false): Promise<voi
 
     const databaseEmulator = new DatabaseEmulator(args);
     await startEmulator(databaseEmulator);
+
+    if (exportMetadata.database) {
+      const importDirAbsPath = path.resolve(options.import);
+      const databaseExportDir = path.join(importDirAbsPath, exportMetadata.database.path);
+
+      const files = fs.readdirSync(databaseExportDir);
+      for (const f of files) {
+        const fPath = path.join(databaseExportDir, f);
+        const ns = f.split(".json")[0];
```

Comment:
nit: Please don't do this. It's rare, but completely possible, for people to create a directory called `i.love.json.files` and put their project there. Use [`path.basename(f, '.json')`](https://nodejs.org/api/path.html#path_path_basename_path_ext) instead.

Reply:
While we're at it, please also filter out files that are not `.json`. There are some operating systems that drop internal files and previews in directories...